docs/src/dev-guide/coding-guidelines.md

```
@@ -53,6 +53,110 @@ To avoid including a state variable in a React Hook's dependency array, you can
+* Use `updateXxx` for actions that do more than simply changing a value (e.g., perform additional logic, make API calls,
+update multiple states).
+
+## Feature-based slicing
+
+When a Zustand store file becomes too large, we should slice it based on features. Avoid grouping by type (e.g., all
+values / actions in one object) — it's a common anti-pattern.
+
+## Store Usage
+
+There are three ways to access Zustand store values and actions:
+* `get() & set()`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)